modules/node_modules/@ncigdc/components/FacetSelection.js

```diff
@@ -0,0 +1,207 @@
+      setQuery('');
+      setFocusedFacet(null);
+      onRequestClose();
+    },
+  }),
+  withHandlers({
+    handleKeyDown: ({ filteredFacets, focusedFacet, setFocusedFacet, handleSelectFacet, handleClose }) => (event) => ({
+      ArrowUp: () => {
+        event.preventDefault();
+        if (_.isEmpty(filteredFacets)) return;
+        console.log(getPreviousItem(filteredFacets, focusedFacet));
+        setFocusedFacet(getPreviousItem(filteredFacets, focusedFacet));
+      },
+      ArrowDown: () => {
+        event.preventDefault();
+        if (_.isEmpty(filteredFacets)) return;
+        setFocusedFacet(getNextItem(filteredFacets, focusedFacet));
+      },
+      Enter: () => handleSelectFacet(focusedFacet),
+      Escape: () => handleClose(),
+    }[event.key] || _.noop)(),
+  }),
+  lifecycle({
+    componentDidMount(): void {
+      this.props.fetchData();
+    },
+  })
+)((props) => (
+  <div>
+    <div {...css(styles.header)}>
+      <h2 data-translate className="modal-title">
+        <span>{props.title}</span>
+        <a
+          onClick={props.handleClose} className="pull-right" style={{ fontSize: '1.5rem' }}
+        >Cancel</a>
+      </h2>
+      <div style={{ marginBottom: 15 }}>
+        <label htmlFor="quick-search-input">Search for a field:</label>
+        <input
+          type="text"
+          autoComplete="off"
+          autoFocus
+          className="form-control"
+          placeholder="search"
+          defaultValue={props.query}
+          onChange={props.handleQueryInputChange}
+          onKeyDown={props.handleKeyDown}
+        />
+      </div>
+      <h3 {...css(styles.resultsCount)}>
+        { props.isLoading ? 'Loading...' : `${props.filteredFacets.length} cases fields` }
+      </h3>
+
+      <label tabIndex={0} role="button" className="pull-right" style={{ display: 'none' }}>
+        <input type="checkbox" />
+        Only show fields with values
+      </label>
+
+    </div>
+
+    <ul {...css(styles.facetList)}>
+      {
+        _.map(props.filteredFacets, (facet) => {
+          const isFocused = props.focusedFacet && facet.full === props.focusedFacet.full;
+          return (
+            <li
+              key={facet.full}
+              onClick={() => props.handleSelectFacet(facet)}
+              onMouseEnter={() => props.setFocusedFacet(facet)}
+              {...css(styles.facetItem)}
+            >
+              <div {...css(styles.itemIconWrapper)}>
+                <i {...css(styles.itemIcon)} className={`fa ${getIconClassNameByEntityType(({ cases: 'case', files: 'file' }[facet.doc_type]))}`} />
+              </div>
+              <div {...css(styles.facetTexts, isFocused && styles.focusedItem.container)}>
+                <span {...css(styles.facetTitle, isFocused && styles.focusedItem.text)}>
+                  <ConditionalHighlight condition={props.query.length >= 2} search={props.query}>{ facet.field }</ConditionalHighlight>
```

Comment:
<img width="353" alt="screen shot 2017-02-16 at 10 33 01 am" src="https://cloud.githubusercontent.com/assets/1314446/23027753/5344beb2-f433-11e6-970a-b4d17a40d34c.png">
to match prod, add the type of the field and italicize the description